src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/ImportEmailTemplatesActivity/Global/OrderReturnCreated/Contents/Html/en-CA.html

```diff
@@ -148,7 +148,7 @@ <h1 style="font-size: 32px;">Hi {{Return.CustomerFirstName}} {{Return.CustomerLa
                                             <!--{% for additionalFee in Return.AggregatedAdditionalFees %}-->
                                             <tr>
                                                 <td colspan="3">&nbsp;</td>
-                                                <td style="line-height: 20px; text-align: left;" colspan="2">{{additionalFee.DisplayName | localize:"en-CA"}}:</td>
+                                                <td style="line-height: 20px; text-align: left;" colspan="2"><!--{% if additionalFee.DisplayName.en-CA !='' and  additionalFee.DisplayName.en-CA !=  null  %}-->{{additionalFee.DisplayName | localize:"en-CA"}}<!--{% else %}--> {{additionalFee.Name}} <!--{% endif %}-->:</td>
```

Comment:
can we try to look to code which renders this and see the ability to have a fallback value?
example of  usage or probably some other format, need to look
{{additionalFee.DisplayName | localize:"en-CA" fallback: additionalFee.Name}}